extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -255,9 +261,10 @@ export abstract class AzureAuth implements vscode.Disposable {
 	}
 
 	public async getToken(tenant: Tenant, resource: Resource, postData: AuthorizationCodePostData | TokenPostData | RefreshTokenPostData): Promise<OAuthTokenResponse> {
+		Logger.verbose('Fetching token');
 		const tokenUrl = `${this.loginEndpointUrl}${tenant.id}/oauth2/token`;
 		const response = await this.makePostRequest(tokenUrl, postData);
-
+		Logger.pii(`Token: `, [response.data], []);
```

Comment:
What are the arrays for? Is there any reason you couldn't just have this?

```
`Token: ${response.data}`
```

(also nit - but if you aren't actually using template strings you shouldn't use the backticks and instead just use 's)

Reply:
This is because the response data needs to be sanitized before it's put into the final output string

Reply:
Ah shoot, yeah sorry I forgot about that. Looks good!